tools/sdk/uxdProtocol/createEditMercurialVaultDepositoryInstruction.ts

```diff
@@ -56,7 +56,7 @@ const createEditMercurialVaultDepositoryInstruction = async ({
       mintingFeeInBps,
       redeemingFeeInBps,
       mintingDisabled,
-      profitsBeneficiaryCollateral: new PublicKey(profitsBeneficiaryCollateral ?? ""),
```

Comment:
this was wrong, if the `profitsBeneficiaryCollateral` was not specified, could not create the proposal